src/main/scala/com/github/unchama/seichiassist/infrastructure/scalikejdbc/ScalikeJDBCConfiguration.scala

```diff
@@ -5,7 +5,12 @@ import scalikejdbc._
 object ScalikeJDBCConfiguration {
 
   private val connectionPoolSettings: ConnectionPoolSettings =
-    ConnectionPoolSettings(initialSize = 5, maxSize = 20, connectionTimeoutMillis = 100000L)
+    ConnectionPoolSettings(
+      initialSize = 5,
+      maxSize = 20,
+      connectionTimeoutMillis = 100000L,
+      driverName = "org.mariadb.jdbc.Driver"
```

Comment:
これは relocate 後の FQCN が入るべきです (他プラグインが導入していない限り `org.mariadb.jdbc.Driver` は実行時に存在しません)。

```suggestion
      driverName = "com.github.unchama.seichiassist.relocateddependencies.org.mariadb.jdbc.Driver"
```